jni/include/faiss_stream_support.h

```diff
@@ -56,6 +56,40 @@ class FaissOpenSearchIOReader final : public faiss::IOReader {
 };  // class FaissOpenSearchIOReader
 
 
+/**
+ * A glue component inheriting IOWriter to delegate IO processing down to the given
+ * mediator. The mediator is expected to do write bytes via the provided Lucene's IndexOutput.
+ */
+class FaissOpenSearchIOWriter final : public faiss::IOWriter {
```

Comment:
If FaissOpenSearchIOWriter ever needs to release resources or perform cleanup, could explicitly define a destructor?Even if the mediator is not owned, a comment can clarify this behavior.

Reply:
`FaissOpenSearchIOWriter` does not have a resource to be released!
It just has a raw pointer

Reply:
Sure